spec.html

```diff
@@ -15552,18 +15574,25 @@ <h1>Runtime Semantics: PropertyDefinitionEvaluation</h1>
         <emu-alg>
           1. Let _propKey_ be the result of evaluating |PropertyName|.
           1. ReturnIfAbrupt(_propKey_).
-          1. If IsAnonymousFunctionDefinition(|AssignmentExpression|) is *true*, then
+          1. If this |PropertyDefinition| is contained within a |Script| which is being evaluated for JSON.parse (see step <emu-xref href="#step-json-parse-parse"></emu-xref> of <emu-xref href="#sec-json.parse">JSON.parse</emu-xref>), then
+            1. Let _isProtoSetter_ be *false*.
+          1. Else if _propKey_ is the String value *"__proto__"* and if IsComputedPropertyKey(|PropertyName|) is *false*, then
```

Comment:
It's unusual to invoke an SDO via prefix-parenthesized syntax (although it predates this PR).
```suggestion
          1. Else if _propKey_ is the String value *"__proto__"* and if IsComputedPropertyKey of |PropertyName| is *false*, then
```